ingestion/src/metadata/ingestion/source/database/mssql/metadata.py

```diff
@@ -27,10 +31,33 @@
 from metadata.utils import fqn
 from metadata.utils.filters import filter_by_database
 from metadata.utils.logger import ingestion_logger
+from metadata.utils.sql_queries import MSSQL_GET_TABLE_COMMENTS
 
 logger = ingestion_logger()
 
 
+@reflection.cache
+def get_table_comment(  # pylint: disable=unused-argument
+    self, connection, table_name, schema_name, **kw
+):
+    """
+    Returns comment of table.
+    """
+    cursor = connection.execute(
+        MSSQL_GET_TABLE_COMMENTS.format(schema_name=schema_name, table_name=table_name)
+    )
+    try:
+        for result in cursor:
+            if result[1]:
+                return {"text": result[1]}
+    except Exception:
+        return {"text": None}
+    return {"text": None}
```

Comment:
This could be simplified to something like

```python
    try:
        for result in cursor:
            if result[1]:
                return {"text": result[1]}
    except Exception as exc:
        logger.error("something happened blabla")

    return {"text": None}
```

Reply:
No need to have so many returns. Also, maybe instead of logger.error a logger.debug could be useful here to know if we are missing descriptions for a valid reason 🤔

Reply:
Sounds good @pmbrull 
i will do that.